### PR TITLE
Give Boskos More Resources

### PR DESCRIPTION
**Pull request by upodroid**
**Which issue(s) this PR fixes**:<br>
Reduced replica count but made the resource limits much higher.

### CODE REVIEW

File: prow/cluster/build/400-boskos-janitor.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ metadata:
     app: boskos-janitor
   namespace: test-pods
 spec:
-  replicas: 6  # Number of distributed janitor instances; k8s has ~1/16 as many projects
+  replicas: 4  # Number of distributed janitor instances; k8s has ~1/16 as many projects
```
Comment (chizhg) [inline prow/cluster/build/400-boskos-janitor.yaml:24]: Why reducing this number?

File: prow/cluster/build/400-boskos-janitor.yaml [hunk 2]
```diff
@@ -47,11 +47,11 @@ spec:
           readOnly: true
         resources:
           requests:
-            cpu: 200m
-            memory: 20Mi
+            cpu: 2000m
```
Comment (chizhg) [inline prow/cluster/build/400-boskos-janitor.yaml:50]: This is likely more than enough. For now let's keep the requests and only bump up the limits, and see how it goes?

```
requests:
    cpu: 200m
    memory: 20Mi
limits:
    cpu: 2000m
    memory: 2560Mi
```

File: prow/cluster/build/400-boskos-janitor.yaml (hunk 1 above)
Comment (upodroid) [reply prow/cluster/build/400-boskos-janitor.yaml:24]: I'm thinking fewer replicas but larger ones. I can revert it if it won't work.

File: prow/cluster/build/400-boskos-janitor.yaml (hunk 2 above)
Comment (upodroid) [reply prow/cluster/build/400-boskos-janitor.yaml]: Fixed

**Pull request merged**